pkg/component/openvpn.go

```diff
@@ -47,11 +57,45 @@ func (o OpenVPN) Configure(overwrite bool) error {
 }
 
 func (o OpenVPN) Init(dir string) error {
-	ca, privateKey, err := pki.NewCertificateAuthority(&CertConfig)
+	now := time.Now()
+	privateKey, err := rsa.GenerateKey(rand.Reader, 2048)
+	if err != nil {
+		log.Fatal(err)
+	}
+	tmpl := x509.Certificate{
+		SerialNumber: new(big.Int).SetInt64(0),
+		Subject: pkix.Name{
+			CommonName:   "openvpn",
+			Organization: []string{"SIGHUP s.r.l."},
+		},
+		NotBefore:             now.UTC(),
+		NotAfter:              now.Add(time.Hour * 24 * 365 * 10).UTC(),
```

Comment:
Should we document that the default (static) duration for the certificate is 10 years?

Reply:
This is "just" the CA, `kubeadm` creates the CAs with the same duration.